src/main/java/teammates/ui/webapi/EnrollStudentsAction.java

```diff
@@ -71,6 +72,7 @@ JsonResult execute() {
         Set<String> existingStudentsEmail =
                 existingStudents.stream().map(StudentAttributes::getEmail).collect(Collectors.toSet());
         List<StudentAttributes> enrolledStudents = new ArrayList<>();
+        List<EnrollStudentsData.EnrollErrorResults> failToEnrollStudents = new ArrayList<>();
```

Comment:
Nit: Feels weird to start a non-function variable name with a verb. Suggest making this a noun. Something like `enrolledUnsuccessfulStudents` (not the best example) or something better.